clang/unittests/Format/FormatTest.cpp

```diff
@@ -19056,6 +19056,9 @@ TEST_F(FormatTest, AlignConsecutiveDeclarations) {
   verifyFormat("int    a(int x);\n"
                "double b();",
                Alignment);
+  verifyFormat("int    a(const Test & = Test());\n"
```

Comment:
I'd like to see something like
``` c++
int foo(someType & whatever, differentType & foo,
        Test & = Test());
```
Maybe in some other variants, I want to see the alignment of the arguments.

Reply:
I wanted to see a version where the arguments are aligned, you need a break in the argument list. Either by providing more arguments, or (preferred) by using a reduced `ColumnLimit` for that test.

Reply:
Got it, I have added a new test to cover this use case.